static/app/views/replays/types.tsx

```diff
@@ -2,6 +2,37 @@ import type {eventWithTime} from 'rrweb/typings/types';
 
 import type {RawCrumb} from 'sentry/types/breadcrumbs';
 
+// Keep this in sync with the backend blueprint
+// "ReplayRecord" is distinct from the common: "replay = new ReplayReader()"
```

Comment:
should we rename these cases to be `replayReader` instead?

Reply:
that's a lot more renames :(

I was also thinking i could split out the `startTimestampMS` renames so that is more of a mechanical PR vs. the value changes. (should it be `startTimestampMs`, zomg i messed it up)